packages/client/hmi-client/src/components/drilldown/tera-drilldown-preview.vue

```diff
@@ -16,30 +16,22 @@
 			</main>
 		</div>
 		<footer>
-			<Button
-				v-if="canSaveAsset"
-				outlined
-				label="Save Model"
-				@click="emit('save-asset')"
-				class="save-asset-button"
-			></Button>
-			<Button outlined label="Cancel" @click="emit('cancel')"></Button>
-			<Button label="Apply Changes and Close" @click="emit('apply-changes')"></Button>
+			<slot name="footer" />
```

Comment:
I don't think this footer is as standardized as we first made it out to be - I'm giving the user an option to add their own